schema/library.json

```diff
@@ -1558,5 +1558,29 @@
             "        pass",
             "end"
         ]
+    },
+    "VAVTurndown_Using_Average": {
+    "library_item_id": 68,
+    "description_brief": "When a VAV box is in reheat mode, the ratio of V_dot_VAV to V_dot_VAV_max should be higher than when it isn't in reheat mode",
+    "description_index": [
+       "Section 6.5.2.1 ASHRAE 90.1-2016"
+    ],
+    "description_datapoints": {
+       "reheat_coil_flag": "VAV box reheat coil operation status",
+       "V_dot_VAV": "VAV airflow rate",
+       "V_dot_VAV_max": "VAV max airflow rate"
+    },
+    "description_verification_type": "procedure-based",
+    "assertions_type": "pass",
+    "description_assertions": [
+       "V_dot_VAV_ratio = V_dot_VAV/V_dot_VAV_max",
+       "mean_reheat_ratio = df.loc[self.df[`reheat_coil_flag`], `V_dot_VAV_ratio`].mean()",
+       "mean_no_reheat_ratio = df.loc[~self.df[`reheat_coil_flag`], `V_dot_VAV_ratio`].mean()",
+       "if mean_reheat_ratio >= mean_no_reheat_ratio:",
+       "    pass",
+       "else: ",
+       "    fail",
+       "end"
```

Comment:
Maybe let's also add the `Untested` case.

Reply:
Sure. Adddressed.